settings.coffee

```diff
@@ -34,7 +34,7 @@ settings =
 	# Documentation about the URL connection string format can be found at:
 	#
 	#    http://docs.mongodb.org/manual/reference/connection-string/
-	# 
```

Comment:
unrelated whitespace changes

`$ git checkout upstream/master settings.coffee`

Reply:
I just saw the update of the PR body.